fmt/printf.h

```diff
@@ -588,6 +600,8 @@ FMT_VARIADIC(int, printf, CStringRef)
   \endrst
  */
 inline int fprintf(std::ostream &os, CStringRef format_str, ArgList args) {
+  std::mutex mtx;
+  std::unique_lock<std::mutex> lck(mtx);
```

Comment:
I don't think this is necessary because the writer is not accessible from multiple threads and it doesn't use any global mutable state.